Docs: add YouTube video link and description (#85484)
grafana/grafana: docs/sources/panels-visualizations/visualizations/logs/index.md

To limit the number of lines rendered, you can use the **Max data points** setting in the **Query options**. If it is not set, then the data source will usually enforce a default limit.

> The following video provides a walkthrough of creating a logs visualization. You'll also learn how to customize some settings and some log visualization caveats:

Reviewer: ```suggestion
The following video provides a walkthrough of creating a logs visualization. You'll also learn how to customize some settings and log visualization caveats:
```

> In the following video you can watch a walkthrough creating a Log Panel Visualization. You will learn as well how to customize a few settings and some Log Panel caveats, check out the video:

Reviewer: ```suggestion
The following video provides a walkthrough of creating a logs visualization. You'll also learn how to customize some settings and some log visualization caveats:
```
